src/legacy/core_plugins/kibana/public/dashboard/__tests__/get_saved_dashboard_mock.ts

```diff
@@ -17,7 +17,8 @@
  * under the License.
  */
 
-import { searchSourceMock } from 'ui/courier/search_source/mocks';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { searchSourceMock } from '../../../../../../plugins/data/public/search/search_source/mocks';
```

Comment:
What's the recommended way of handling that in the future? Will it be added to the top level `mocks` export of the data plugin? Should we create our own mock?

Reply:
I think we should add it to the top level mocks - indeed.

https://github.com/elastic/kibana/issues/56001